tests/pds3file/test_pds3file_whitebox.py

```
@@ -323,7 +322,7 @@ def test_data_set_id_exception(self, input_path, expected):
         """lid: return self._lid_filled"""
         target_pdsfile = instantiate_target_pdsfile(input_path)
         try:
-            res1 = target_pdsfile.data_set_id
+            _ = target_pdsfile.data_set_id
         except ValueError as e:
             assert expected in str(e)
```

Ruff (0.16.1)

[warning] Found assertion on exception `e` in `except` block, use `pytest.raises()` instead

(PT017)